pkg/routes/ui.go

```diff
@@ -22,14 +22,18 @@ import (
 	"k8s.io/apiserver/pkg/server/mux"
 )
 
-const dashboardPath = "/api/v1/namespaces/kube-system/services/kubernetes-dashboard/proxy"
-
 // UIRedirect redirects /ui to the kube-ui proxy path.
-type UIRedirect struct{}
+type UIRedirect struct {
+	DashboardPath string
```

Comment:
`DestinationURL`

Reply:
Actually `destinationURL` since it doesn't need to be public.

Reply:
Done